modules/oxide/src/less/theme/components/dialog/dialog.less

```
@@ -238,7 +238,7 @@
     display: flex;
     flex: 1;
     flex-direction: column;
-    max-height: ~"min(650px, calc(100vh - 110px))"; // TODO (verify max-height for dialogs)
+    max-height: ~"min(650px, calc(100vh - 110px))";
```

```suggestion
    max-height: ~"min(650px, calc(100vh - 110px))"; // TODO (verify max-height for dialogs)
```
If this is not implemented as part of this PR then this comment should not be removed.

---

test for dialog(body) height is implemented with this pr